sdk/process/msgs.go

```diff
@@ -0,0 +1,92 @@
+package servicesdk
+
+import (
+	cosmostypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/mesg-foundation/engine/codec"
+	"github.com/mesg-foundation/engine/hash"
+	"github.com/mesg-foundation/engine/protobuf/api"
+)
+
+// msgCreateProcess defines a state transition to create a service.
+type msgCreateProcess struct {
+	Owner   cosmostypes.AccAddress    `json:"owner"`
```

Comment:
you could directly add the validation like on PR https://github.com/mesg-foundation/engine/pull/1519 on `msgCreateProcess` and `msgDeleteProcess`